pages/pil/dashboard/index.js

```diff
@@ -104,17 +106,14 @@ module.exports = settings => {
 
   app.post('/', (req, res, next) => {
     sendData(req)
-      .then(() => delete req.session.form[req.model.id])
-      .then(() => res.redirect(req.buildRoute('pil.update', { suffix: 'success' })))
+      .then(() => {
+        delete req.session.form[req.model.id];
+        return res.redirect(req.buildRoute('pil.update', { suffix: 'success' }));
+      })
```

Comment:
does this need to do the task/session stuff too?

Reply:
Anywhere we have a `sendData()` function I've added it there, because in some places we wrap `sendData()` in a call to `updateDataFromTask()` and I was having issues with saving the task id to the session in those cases.

Reply:
carry on then 👍